### PR TITLE
dev, v3.0: update cluster configuration doc to support tidb/tikv/pd config

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### Preview:
https://github.com/DanielZhangQD/docs-cn/blob/config-change/dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md
https://github.com/DanielZhangQD/docs-cn/blob/config-change/v3.0/reference/configuration/tidb-in-kubernetes/cluster-configuration.md

### What is the related PR or file link(s)? 

- https://github.com/pingcap/docs/pull/1334
- https://github.com/pingcap/docs-cn/pull/1582
- https://github.com/pingcap/tidb-operator/pull/638

### CODE REVIEW

**Comment by xiaojingchen** [comment]
this PR please merge after the #1582

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 1]
```diff
@@ -28,13 +28,14 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
 | `discovery.resoureces.requests.cpu` | PD 服务发现组件的 CPU 资源请求 | `80m` |
 | `discovery.resoureces.requests.memory` | PD 服务发现组件的内存资源请求 | `50Mi` |
 | `enableConfigMapRollout` | 是否开启 TiDB 集群自动滚动更新。如果启用，则 TiDB 集群的 ConfigMap 变更时，TiDB 集群自动更新对应组件。该配置只在 tidb-operator v1.0 及以上版本才支持 | `false` |
+| `pd.config` | 配置文件格式的 PD 的 配置，请参考[链接](https://github.com/pingcap/pd/blob/master/conf/config.toml)查看默认 PD 配置文件（选择对应 PD 版本的 tag），可以参考[文档](https://pingcap.com/docs-cn/v3.0/reference/configuration/pd-server/configuration-file/)查看配置参数的具体介绍（请选择对应的文档版本），这里只需要**按照配置文件中的格式配置要修改的配置** | TiDB Operator 版本 <= v1.0.0-beta.3，默认值为:<br>`nil`<br>TiDB Operator 版本 > v1.0.0-beta.3，默认值为:<br>`[log]`<br>`level = "info"`<br>`[replication]`<br>`location-labels = ["region", "zone", "rack", "host"]`<br>配置示例：<br>&nbsp;&nbsp;`config: |`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[log]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`level = "info"`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[replication]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`location-labels = ["region", "zone", "rack", "host"]` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `pd.config` | 配置文件格式的 PD 的配置，请参考[链接](https://github.com/pingcap/pd/blob/master/conf/config.toml)；查看默认 PD 配置文件（选择对应 PD 版本的 tag），可以参考[文档](https://pingcap.com/docs-cn/v3.0/reference/configuration/pd-server/configuration-file/)查看配置参数的具体介绍（请选择对应的文档版本），这里只需要**按照配置文件中的格式修改配置** | TiDB Operator 版本 <= v1.0.0-beta.3，默认值为 `nil`；<br>TiDB Operator 版本 > v1.0.0-beta.3，默认值为 `[log]`；<br>`level = "info"`<br>`[replication]`<br>`location-labels = ["region", "zone", "rack", "host"]`<br>配置示例：<br>&nbsp;&nbsp;`config:`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[log]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`level = "info"`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[replication]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`location-labels = ["region", "zone", "rack", "host"]` |
```

File: v3.0/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 2]
```diff
@@ -28,13 +28,14 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
 | `discovery.resoureces.requests.cpu` | PD 服务发现组件的 CPU 资源请求 | `80m` |
 | `discovery.resoureces.requests.memory` | PD 服务发现组件的内存资源请求 | `50Mi` |
 | `enableConfigMapRollout` | 是否开启 TiDB 集群自动滚动更新。如果启用，则 TiDB 集群的 ConfigMap 变更时，TiDB 集群自动更新对应组件。该配置只在 tidb-operator v1.0 及以上版本才支持 | `false` |
+| `pd.config` | 配置文件格式的 PD 的 配置，请参考[链接](https://github.com/pingcap/pd/blob/master/conf/config.toml)查看默认 PD 配置文件（选择对应 PD 版本的 tag），可以参考[文档](https://pingcap.com/docs-cn/v3.0/reference/configuration/pd-server/configuration-file/)查看配置参数的具体介绍（请选择对应的文档版本），这里只需要**按照配置文件中的格式配置要修改的配置** | TiDB Operator 版本 <= v1.0.0-beta.3，默认值为:<br>`nil`<br>TiDB Operator 版本 > v1.0.0-beta.3，默认值为:<br>`[log]`<br>`level = "info"`<br>`[replication]`<br>`location-labels = ["region", "zone", "rack", "host"]`<br>配置示例：<br>&nbsp;&nbsp;`config: |`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[log]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`level = "info"`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[replication]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`location-labels = ["region", "zone", "rack", "host"]` |
```
Comment (yikeke) [inline v3.0/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `pd.config` | 配置文件格式的 PD 的配置，请参考[链接](https://github.com/pingcap/pd/blob/master/conf/config.toml)；查看默认 PD 配置文件（选择对应 PD 版本的 tag），可以参考[文档](https://pingcap.com/docs-cn/v3.0/reference/configuration/pd-server/configuration-file/)查看配置参数的具体介绍（请选择对应的文档版本），这里只需要**按照配置文件中的格式修改配置** | TiDB Operator 版本 <= v1.0.0-beta.3，默认值为 `nil`；<br>TiDB Operator 版本 > v1.0.0-beta.3，默认值为 `[log]`；<br>`level = "info"`<br>`[replication]`<br>`location-labels = ["region", "zone", "rack", "host"]`<br>配置示例：<br>&nbsp;&nbsp;`config:`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[log]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`level = "info"`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[replication]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`location-labels = ["region", "zone", "rack", "host"]` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 3]
```diff
@@ -28,13 +28,14 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
 | `pd.replicas` | PD 的 Pod 数 | `3` |
 | `pd.image` | PD 镜像 | `pingcap/pd:v3.0.0-rc.1` |
 | `pd.imagePullPolicy` | PD 镜像的拉取策略 | `IfNotPresent` |
-| `pd.logLevel` | PD 日志级别 | `info` |
+| `pd.logLevel` | PD 日志级别<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `pd.config` 配置:<br>`[log]`<br>`level = "info"` | `info` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `pd.logLevel` | PD 日志级别<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `pd.config` 配置：<br>`[log]`<br>`level = "info"` | `info` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 4]
```diff
@@ -28,13 +28,14 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
 | `pd.storageClassName` | PD 使用的 storageClass，storageClassName 指代一种由 Kubernetes 集群提供的存储类型，不同的类可能映射到服务质量级别、备份策略或集群管理员确定的任意策略。详细参考：[storage-classes](https://kubernetes.io/docs/concepts/storage/storage-classes) | `local-storage` |
-| `pd.maxStoreDownTime` | `pd.maxStoreDownTime` 指一个 store 节点断开连接多长时间后状态会被标记为 `down`，如果状态变为 `down` 后，store 节点开始迁移数据到其它 store 节点 | `30m` |
-| `pd.maxReplicas` | `pd.maxReplicas` 是 TiDB 集群的数据的副本数 | `3` |
+| `pd.maxStoreDownTime` | `pd.maxStoreDownTime` 指一个 store 节点断开连接多长时间后状态会被标记为 `down`，如果状态变为 `down` 后，store 节点开始迁移数据到其它 store 节点<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `pd.config` 配置:<br>`[schedule]`<br>`max-store-down-time = "30m"` | `30m` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `pd.maxStoreDownTime` | `pd.maxStoreDownTime` 指一个 store 节点断开连接多长时间后状态会被标记为 `down`，当状态变为 `down` 后，store 节点开始迁移数据到其它 store 节点<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `pd.config` 配置：<br>`[schedule]`<br>`max-store-down-time = "30m"` | `30m` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 5]
```diff
@@ -28,13 +28,14 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
+| `pd.maxReplicas` | `pd.maxReplicas` 是 TiDB 集群的数据的副本数<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `pd.config` 配置:<br>`[replication]`<br>`max-replicas = 3` | `3` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `pd.maxReplicas` | `pd.maxReplicas` 是 TiDB 集群的数据的副本数<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `pd.config` 配置：<br>`[replication]`<br>`max-replicas = 3` | `3` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 6]
```diff
@@ -45,12 +46,14 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
 | `pd.nodeSelector` | `pd.nodeSelector` 确保 PD Pods 只调度到以该键值对作为标签的节点，详情参考：[nodeselector](https://kubernetes.io/docs/concepts/configuration/assign-Pod-node/#nodeselector) | `{}` |
 | `pd.tolerations` | `pd.tolerations` 应用于 PD Pods，允许 PD Pods 调度到含有指定 taints 的节点上，详情参考：[taint-and-toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration) | `{}` |
 | `pd.annotations` | 为 PD Pods 添加特定的 `annotations` | `{}` |
+| `tikv.config` | 配置文件格式的 TiKV 的 配置，请参考[链接](https://github.com/tikv/tikv/blob/master/etc/config-template.toml)查看默认 TiKV 配置文件（选择对应 TiKV 版本的 tag），可以参考[文档](https://pingcap.com/docs-cn/v3.0/reference/configuration/tikv-server/configuration-file/)查看配置参数的具体介绍（请选择对应的文档版本），这里只需要**按照配置文件中的格式配置要修改的配置** | TiDB Operator 版本 <= v1.0.0-beta.3，默认值为:<br>`nil`<br>TiDB Operator 版本 > v1.0.0-beta.3，默认值为:<br>`log-level = "info"`<br>配置示例：<br>&nbsp;&nbsp;`config: |`<br>&nbsp;&nbsp;&nbsp;&nbsp;`log-level = "info"` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `tikv.config` | 配置文件格式的 TiKV 的 配置，请参考[链接](https://github.com/tikv/tikv/blob/master/etc/config-template.toml)查看默认 TiKV 配置文件（选择对应 TiKV 版本的 tag），可以参考[文档](https://pingcap.com/docs-cn/v3.0/reference/configuration/tikv-server/configuration-file/)查看配置参数的具体介绍（请选择对应的文档版本），这里只需要**按照配置文件中的格式配置要修改的配置** | TiDB Operator 版本 <= v1.0.0-beta.3，默认值为 `nil`<br>TiDB Operator 版本 > v1.0.0-beta.3 时，默认值为 `log-level = "info"`<br>配置示例：<br>&nbsp;&nbsp;`config:`<br>&nbsp;&nbsp;&nbsp;&nbsp;`log-level = "info"` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 7]
```diff
@@ -45,12 +46,14 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
 | `tikv.replicas` | TiKV 的 Pod 数 | `3` |
 | `tikv.image` | TiKV 的镜像 | `pingcap/tikv:v3.0.0-rc.1` |
 | `tikv.imagePullPolicy` | TiKV 镜像的拉取策略 | `IfNotPresent` |
-| `tikv.logLevel` | TiKV 的日志级别 | `info` |
+| `tikv.logLevel` | TiKV 的日志级别<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置:<br>`log-level = "info"` | `info` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `tikv.logLevel` | TiKV 的日志级别<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置：<br>`log-level = "info"` | `info` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 8]
```diff
@@ -45,12 +46,14 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
 | `tikv.storageClassName` | TiKV 使用的 storageClass，storageClassName 指代一种由 Kubernetes 集群提供的存储类型，不同的类可能映射到服务质量级别、备份策略或集群管理员确定的任意策略。详细参考：[storage-classes](https://kubernetes.io/docs/concepts/storage/storage-classes) | `local-storage` |
-| `tikv.syncLog` | syncLog 指是否启用 raft 日志同步功能，启用该功能能保证在断电时数据不丢失 | `true` |
+| `tikv.syncLog` | syncLog 指是否启用 raft 日志同步功能，启用该功能能保证在断电时数据不丢失<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置:<br>`[raftstore]`<br>`sync-log = true` | `true` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `tikv.syncLog` | syncLog 指是否启用 raft 日志同步功能，启用该功能能保证在断电时数据不丢失<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置：<br>`[raftstore]`<br>`sync-log = true` | `true` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 9]
```diff
@@ -45,12 +46,14 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
+| `tikv.grpcConcurrency` | 配置 gRPC server 线程池大小<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置:<br>`[server]`<br>`grpc-concurrency = 4` | `4` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `tikv.grpcConcurrency` | 配置 gRPC server 线程池大小<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置：<br>`[server]`<br>`grpc-concurrency = 4` | `4` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 10]
```diff
@@ -61,15 +64,16 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
 | `tikv.annotations` | 为 TiKV Pods 添加特定的 `annotations` | `{}` |
-| `tikv.defaultcfBlockCacheSize` | 指定 block 缓存大小，block 缓存用于缓存未压缩的 block，较大的 block 缓存设置可以加快读取速度。一般推荐设置为 `tikv.resources.limits.memory` 的 30%-50% | `1GB` |
-| `tikv.writecfBlockCacheSize` | 指定 writecf 的 block 缓存大小，一般推荐设置为 `tikv.resources.limits.memory` 的 10%-30% | `256MB` |
-| `tikv.readpoolStorageConcurrency` | TiKV 存储的高优先级/普通优先级/低优先级操作的线程池大小 | `4` |
-| `tikv.readpoolCoprocessorConcurrency` | 一般如果 `tikv.resources.limits.cpu` > 8，则 `tikv.readpoolCoprocessorConcurrency` 设置为`tikv.resources.limits.cpu` * 0.8 | `8` |
-| `tikv.storageSchedulerWorkerPoolSize` | TiKV 调度程序的工作池大小，应在重写情况下增加，同时应小于总 CPU 核心 | `4` |
+| `tikv.defaultcfBlockCacheSize` | 指定 block 缓存大小，block 缓存用于缓存未压缩的 block，较大的 block 缓存设置可以加快读取速度。一般推荐设置为 `tikv.resources.limits.memory` 的 30%-50%<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置:<br>`[rocksdb.defaultcf]`<br>`block-cache-size = "1GB"`<br>从 TiKV v3.0.0 开始，不再需要配置 `[rocksdb.defaultcf].block-cache-size` 和 `[rocksdb.writecf].block-cache-size`，改为配置 `[storage.block-cache].capacity` | `1GB` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `tikv.defaultcfBlockCacheSize` | 指定 block 缓存大小，block 缓存用于缓存未压缩的 block，较大的 block 缓存设置可以加快读取速度。一般推荐设置为 `tikv.resources.limits.memory` 的 30%-50%<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置：<br>`[rocksdb.defaultcf]`<br>`block-cache-size = "1GB"`<br>从 TiKV v3.0.0 开始，不再需要配置 `[rocksdb.defaultcf].block-cache-size` 和 `[rocksdb.writecf].block-cache-size`，改为配置 `[storage.block-cache].capacity` | `1GB` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 11]
```diff
@@ -61,15 +64,16 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
+| `tikv.writecfBlockCacheSize` | 指定 writecf 的 block 缓存大小，一般推荐设置为 `tikv.resources.limits.memory` 的 10%-30%<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置:<br>`[rocksdb.writecf]`<br>`block-cache-size = "256MB"`<br>从 TiKV v3.0.0 开始，不再需要配置 `[rocksdb.defaultcf].block-cache-size` 和 `[rocksdb.writecf].block-cache-size`，改为配置 `[storage.block-cache].capacity` | `256MB` |
+| `tikv.readpoolStorageConcurrency` | TiKV 存储的高优先级/普通优先级/低优先级操作的线程池大小<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置:<br>`[readpool.storage]`<br>`high-concurrency = 4`<br>`normal-concurrency = 4`<br>`low-concurrency = 4` | `4` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `tikv.readpoolStorageConcurrency` | TiKV 存储的高优先级/普通优先级/低优先级操作的线程池大小<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置：<br>`[readpool.storage]`<br>`high-concurrency = 4`<br>`normal-concurrency = 4`<br>`low-concurrency = 4` | `4` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md (hunk 3 above)
Comment (yikeke) [reply dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]: 建议通过批量替换一下这篇文档类似位置的英文冒号 `:`，比较方便。 @DanielZhangQD

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 12]
```diff
@@ -61,15 +64,16 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
+| `tikv.readpoolCoprocessorConcurrency` | 一般如果 `tikv.resources.limits.cpu` > 8，则 `tikv.readpoolCoprocessorConcurrency` 设置为`tikv.resources.limits.cpu` * 0.8<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置:<br>`[readpool.coprocessor]`<br>`high-concurrency = 8`<br>`normal-concurrency = 8`<br>`low-concurrency = 8` | `8` |
+| `tikv.storageSchedulerWorkerPoolSize` | TiKV 调度程序的工作池大小，应在重写情况下增加，同时应小于总 CPU 核心<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tikv.config` 配置:<br>`[storage]`<br>`scheduler-worker-pool-size = 4` | `4` |
+| `tidb.config` | 配置文件格式的 TiDB 的 配置，请参考[链接](https://github.com/pingcap/tidb/blob/master/config/config.toml.example)查看默认 TiDB 配置文件（选择对应 TiDB 版本的 tag），可以参考[文档](https://pingcap.com/docs-cn/v3.0/reference/configuration/tidb-server/configuration-file/)查看配置参数的具体介绍（请选择对应的文档版本），这里只需要**按照配置文件中的格式配置要修改的配置** | TiDB Operator 版本 <= v1.0.0-beta.3，默认值为:<br>`nil`<br>TiDB Operator 版本 > v1.0.0-beta.3，默认值为:<br>`[log]`<br>`level = "info"`<br>配置示例：<br>&nbsp;&nbsp;`config: |`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[log]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`level = "info"` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `tidb.config` | 配置文件格式的 TiDB 的配置，请参考[链接](https://github.com/pingcap/tidb/blob/master/config/config.toml.example)查看默认 TiDB 配置文件（选择对应 TiDB 版本的 tag），可以参考[文档](https://pingcap.com/docs-cn/v3.0/reference/configuration/tidb-server/configuration-file/)查看配置参数的具体介绍（请选择对应的文档版本），这里只需要**按照配置文件中的格式配置要修改的配置** | TiDB Operator 版本 <= v1.0.0-beta.3，默认值为 `nil`<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，默认值为：<br>`[log]`<br>`level = "info"`<br>配置示例：<br>&nbsp;&nbsp;`config:`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[log]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`level = "info"` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 13]
```diff
@@ -87,7 +91,7 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
 | `tidb.service.mysqlNodePort` | TiDB 服务暴露的 mysql NodePort 端口 |  |
 | `tidb.service.exposeStatus` | TiDB 服务是否暴露状态端口 | `true` |
 | `tidb.service.statusNodePort` | 指定 TiDB 服务的状态端口暴露的 `NodePort` |  |
-| `tidb.separateSlowLog` | 是否以 sidecar 方式运行独立容器输出 TiDB 的 SlowLog | `false` |
+| `tidb.separateSlowLog` | 是否以 sidecar 方式运行独立容器输出 TiDB 的 SlowLog | TiDB Operator 版本 <= v1.0.0-beta.3，默认值为:<br>`false`<br>TiDB Operator 版本 > v1.0.0-beta.3，默认值为:<br>`true` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `tidb.separateSlowLog` | 是否以 sidecar 方式运行独立容器输出 TiDB 的 SlowLog | 如果 TiDB Operator 版本 <= v1.0.0-beta.3，默认值为 `false`<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，默认值为 `true` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 14]
```diff
@@ -96,19 +100,19 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
-| `tidb.tokenLimit` | TiDB 并发执行会话的限制 | `1000` |
-| `tidb.memQuotaQuery` | TiDB 查询的内存限额，默认 32GB | `34359738368` |
-| `tidb.txnEntryCountLimit` | 一个事务中条目的数目限制。如果使用 TiKV 作为存储，则条目表示键/值对。**警告**：不要将该值设置得太大，否则会对 TiKV 集群造成很大影响。请仔细调整此配置 | `300000` |
-| `tidb.txnTotalSizeLimit` | 一个事务中各条目的字节大小限制。如果使用 TiKV 作为存储，则条目表示键/值对。**警告**：不要将该值设置得太大，否则会对 TiKV 集群造成很大影响。请仔细调整此配置 | `104857600` |
-| `tidb.enableBatchDml` | `tidb.enableBatchDml` 为 DML 启用批提交 | `false` |
-| `tidb.checkMb4ValueInUtf8` | `tidb.checkMb4ValueInUtf8` | 用于控制当字符集为utf8时是否检查mb4字符 | `true` |
-| `tidb.treatOldVersionUtf8AsUtf8mb4` | `tidb.treatOldVersionUtf8AsUtf8mb4`用于升级兼容性。设置为`true`将把旧版本的表/列 `utf8` 字符集视为 `utf8mb4` | `true` |
-| `tidb.lease` | `tidb.lease`是 TiDB Schema lease 的期限，对其更改是非常危险的，除非你明确知道可能产生的结果，否则不建议更改。 | `45s` |
-| `tidb.maxProcs` | 最大可使用的 CPU 核数，0 代表机器/Pod 上的 CPU 数量 | `0` |
+| `tidb.preparedPlanCacheEnabled` | 是否启用 TiDB 的 prepared plan 缓存<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tidb.config` 配置:<br>`[prepared-plan-cache]`<br>`enabled = false` | `false` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `tidb.preparedPlanCacheEnabled` | 是否启用 TiDB 的 prepared plan 缓存<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tidb.config` 配置：<br>`[prepared-plan-cache]`<br>`enabled = false` | `false` |
```

File: dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md [hunk 15]
```diff
@@ -96,19 +100,19 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群，TiDB 集群的部署配
...
+| `tidb.tokenLimit` | TiDB 并发执行会话的限制<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tidb.config` 配置:<br>`token-limit = 1000` | `1000` |
+| `tidb.memQuotaQuery` | TiDB 查询的内存限额，默认 32GB<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tidb.config` 配置:<br>`mem-quota-query = 34359738368` | `34359738368` |
+| `tidb.txnEntryCountLimit` | 一个事务中条目的数目限制。如果使用 TiKV 作为存储，则条目表示键/值对。**警告**：不要将该值设置得太大，否则会对 TiKV 集群造成很大影响。请仔细调整此配置<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tidb.config` 配置:<br>`[performance]`<br>`txn-entry-count-limit = 300000` | `300000` |
+| `tidb.txnTotalSizeLimit` | 一个事务中各条目的字节大小限制。如果使用 TiKV 作为存储，则条目表示键/值对。**警告**：不要将该值设置得太大，否则会对 TiKV 集群造成很大影响。请仔细调整此配置<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tidb.config` 配置:<br>`[performance]`<br>`txn-total-size-limit = 104857600` | `104857600` |
+| `tidb.enableBatchDml` | `tidb.enableBatchDml` 为 DML 启用批提交<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tidb.config` 配置:<br>`enable-batch-dml = false` | `false` |
+| `tidb.checkMb4ValueInUtf8` | 用于控制当字符集为utf8时是否检查mb4字符<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tidb.config` 配置:<br>`check-mb4-value-in-utf8 = true` | `true` |
```
Comment (yikeke) [inline dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]:
```suggestion
| `tidb.checkMb4ValueInUtf8` | 用于控制当字符集为 utf8 时是否检查 mb4 字符<br>如果 TiDB Operator 版本 > v1.0.0-beta.3，请通过 `tidb.config` 配置：<br>`check-mb4-value-in-utf8 = true` | `true` |
```

File: v3.0/reference/configuration/tidb-in-kubernetes/cluster-configuration.md (hunk 2 above)
Comment (yikeke) [reply v3.0/reference/configuration/tidb-in-kubernetes/cluster-configuration.md]: Please update all the changes to the v3.0 folder too. @DanielZhangQD

**Pull request merged**